Test/BlockImpTest.m

```diff
@@ -24,17 +24,17 @@ +(struct big)sret;
 
 int main(void)
 {
+	__block Class cls = objc_getClass("Foo");
```

Comment:
This shouldn't need to be `__block`, it should be fine to capture by value.  It it isn't, then we should figure out why not.